### PR TITLE
New landing styles

### PR DESCRIPTION
**Pull request by noahmanger**
## Summary
- Includes styles to support the new data landing page, hero designs, feedback box, etc. 
- Removes old icons that have been replaced by inline SVGs

Will document further once I'm done, but putting it here for visibility.

### CODE REVIEW

File: js/helpers.js [hunk 1]
```diff
@@ -25,6 +25,22 @@ function getWindowWidth() {
...
+}
+
+function isMediumScreen() {
+  if (window.innerWidth >= BREAKPOINTS.MEDIUM) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
```
Comment (adborden) [inline js/helpers.js]: Ah, thanks!

File: scss/components/_cards.scss [hunk 2]
```diff
@@ -137,10 +137,13 @@
       float: left;
       color: $primary-contrast;
       padding: u(1rem);
-      margin: 0;
     }
   }
 
+  .card__title {
+    margin-bottom: 1rem;
```
Comment (adborden) [inline scss/components/_cards.scss]: `u(1rem);`

File: scss/components/_search-bar.scss [hunk 3]
```diff
@@ -37,6 +37,10 @@
   }
 }
 
+.search__example {
+  margin: u(1rem) 0 0 u(21rem);
```
Comment (adborden) [inline scss/components/_search-bar.scss]: :+1: or `u(1rem 0 0 21rem);`

File: scss/modules/_hero.scss [hunk 4]
```diff
@@ -21,72 +21,59 @@
...
+  background-size: cover;
+  height: u(4rem);
+  width: 100%;
+}
 
-  @include media($lg) {
-    @include span-columns(3 of 8);
-    @include shift(2.5);
+.hero__title {
+  margin: 2rem 0 0 0;
```
Comment (adborden) [inline scss/modules/_hero.scss]: `u(2rem 0 0 0);`

Comment (noahmanger) [inline scss/modules/_hero.scss]: 🙈 gah. Force of habit. Thanks.

**Comment by adborden** [comment]
👍 Can you add docs for some of the new subclasses? Like the new button, tooltip, and hero subclasses?

**Pull request merged**